### PR TITLE
Feat/date input

### PR DESCRIPTION
**Pull request by Thienhuynh95**
## Description

This PR:
- Rewrite flow of generating amino constant with cleaner regex flow
- Update test capture, snapshots for the changes
- Rename `snapshot` with `capture` for differentiate jest snapshot
- Add `date` input type into input component
- Color change for error state in Help text component
- Update main `README`

## Todo

- [x] Bump version and add tag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,26 @@
 ## Quick start
 
 ## Documentation
+### - Available features in Amino:
+- [Style generation (generate style constants and css file)](./build-utils/css/README.md)
+- [Icon component generation process (generate optimized react components based on SVG downloaded from Figma/any sources)](./svgReact/icons/README.md)
+- [Flag component generation process (generate optimized react components for flag svgs from smallest svg size of AWS and Figma)](./svgReact/flags/README.md)
```
Comment (goodpickle) [inline README.md:13]: Thank you for updating the Readme 💯

**Review by goodpickle — CHANGES_REQUESTED** [review]
What do you think about adding a DateInput story?

File: README.md (hunk 1 above)
Comment (Thienhuynh95) [reply README.md:13]: I added a story for Date input already. Thanks for catching that!

**Review by evad1n — DISMISSED** [review]

**Review by goodpickle — DISMISSED** [review]
Thank you! 💯

**Comment by devaneereid** [comment]
That's exciting to see the DateInput as an option to use in Amino! Nice job :)

**Review by goodpickle — APPROVED** [review]

File: build-utils/css/utils/__tests__/generateAminoConstants.test.ts [hunk 2]
```diff
@@ -28,18 +28,21 @@ test(`Have and don't have jsdocs comment`, async () => {
   expect(result).toMatchInlineSnapshot(`
```
Comment (ckgardner) [inline build-utils/css/utils/__tests__/generateAminoConstants.test.ts:28]: These snapshot tests are cool! Great idea to test jsdoc deprecations.

**Pull request merged**